integrations/airtable/src/misc/custom-schemas.ts

```diff
@@ -0,0 +1,80 @@
+import z from 'zod'
+import { tableSchema, recordSchema } from './sub-schemas'
+
+export const getTableRecordsInputSchema = z.object({
+  tableIdOrName: z
+    .string()
+    .describe(
+      'The ID or Name of the table (e.g. tblFnqcm4zLVKn85A or articles)'
+    ),
+})
+
+export const getTableRecordsOutputSchema = z
+  .object({
+    records: z.array(recordSchema),
+  })
+  .passthrough()
+
+export const createTableInputSchema = z.object({
+  name: z.string().describe('Name of the Table (e.g. MyTable)'),
+  fields: z
+    .string()
+    .describe(
```

Comment:
**chore:** let's represent this property as it should be: `z.array(z.string())` (don't forget to update UI properties)